src/Backend/Modules/FormBuilder/Js/FormBuilder.js

```diff
@@ -88,6 +91,25 @@ jsBackend.FormBuilder.Fields =
     },
 
     /**
+     * Bind the form submit action
+     */
+    bindFromSubmit: function () {
+        $("#edit").submit(function (e) {
+            // check if a modal window is already open
+            $('.jsFieldDialog').each(function () {
+                // if a modal window is open we prevent the event from propagating
+                if ($(this).css('display') != 'none') {
+                    e.preventDefault();
```

Comment:
`e.preventDefault()` is unnecessary here since `return false` effectively equals `preventDefault` and `stopPropagation`